src/core/templatetags/sidebar.py

```diff
@@ -271,23 +272,44 @@ class UsefulLinks(SidebarPart):
 
     def is_visible(self) -> bool:
         page = self.context.get("self")
-        return bool(getattr(page, "useful_links", None))
+        if not isinstance(page, Page):
+            return False
+
+        useful_links = getattr(page, "useful_links", None)
+        child_pages = page.get_children().exclude(
+            content_type=ContentType.objects.get_for_model(Network)
+        )
+        return bool(useful_links) or bool(child_pages)
```

Comment:
On the last line `return bool(useful_links) or bool(child_pages)` we can cast the whole statement in a bool 😄 

Also, we can make the child_pages db call quicker by performing a count:

```suggestion
        useful_links = getattr(page, "useful_links", None)
        child_pages = page.get_children().exclude(
            content_type=ContentType.objects.get_for_model(Network)
        ).exists()
        return bool(useful_links or child_pages)
```